integration-tests/tests/link-to-new-workspace.js

```diff
@@ -0,0 +1,42 @@
+const { checkbox, click, clickable, clickTableCell, fillIn, waitForNoSpinners } = require('../utils/integration-utils')
+const { enableDataCatalog, testWorkspaceName } = require('../utils/integration-helpers')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const testLinkToNewWorkspaceFn = withUserToken(async ({ testUrl, page, token }) => {
+  await enableDataCatalog(page, testUrl, token)
+  await click(page, clickable({ textContains: 'browse & explore' }))
+  await waitForNoSpinners(page)
+
+  await click(page, checkbox({ text: 'Granted', isDescendant: true }))
+  await clickTableCell(page, "dataset list", 2, 2)
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Link to a workspace' }))
+  await waitForNoSpinners(page)
+
+  const newWorkspaceName = testWorkspaceName()
+  const newWorkspaceBillingAccount = 'general-dev-billing-account'
+  try {
+    await click(page, clickable({ textContains: 'Start with a new workspace' }))
+    await fillIn(page, '//*[@placeholder="Enter a name"]', `${newWorkspaceName}`)
+    await click(page, clickable({ text: 'Select a billing project' }))
+    await click(page, clickable({ text: `${newWorkspaceBillingAccount}` }))
+    await click(page, clickable({ text: 'Create Workspace' }))
+    await waitForNoSpinners(page)
+    await page.url().includes(newWorkspaceName)
+  } finally {
+    await page.evaluate((name, billingProject) => {
+        return window.Ajax().Workspaces.workspace(billingProject, name).delete()
+      }, `${newWorkspaceName}`, `${newWorkspaceBillingAccount}`)
+  }
```

Comment:
There seems to be some mis-indentation above. Could you [run the reformatter](https://github.com/DataBiosphere/terra-ui/wiki/Developing#code-style-and-linting)?

Reply:
The link to the `js-style.xml` file is broken. Is there a different way to configure the code style?

Reply:
@zarsky-broad do you have that file at hand, and/or do you have a different recommendation?

Reply:
Ah, the docs are outdated; intellij should pick up the editorconfig file at the project root and apply it...

Reply:
but you can also just right click and fix eslint problems